std/net/curl.d

```diff
@@ -2879,7 +2879,7 @@ struct HTTP
     /** The headers read from a successful response.
      *
      */
-    @property string[string] responseHeaders()
+    @property string[string] responseHeaders() return
     {
         return p.headersIn;
```

Comment:
>you're not returning a reference to a field, but a reference to the object pointed by the field

Right, that's my issue with this. Because the AA isn't part of the struct and it's GC-ed this is perfectly safe.